### PR TITLE
added validations

### PR DESCRIPTION
**Pull request by ishani220396**
Please merge, I have finished the validations part. Thanks

### CODE REVIEW

File: assessments/adhd.html [hunk 1]
```diff
@@ -89,438 +95,495 @@ <h3 class="mm-survey-results-score"></h3>
...
+                                       </select>
+
+                                       <br>
+                                       <br>
+
+                                       <label for="features"><i style="color: rgb(255, 0, 0); ;">*</i>
+                                          <b>Function/Domain</b></label>
+                                       <select name="domain" id="domain" required>
+                                          <option value="" disabled selected>Select your option</option>
+                                          <option value="Model_S"> Tech</option>
```
Comment (orchvate) [inline assessments/adhd.html:181]: None of these values are compatible with the response structure

File: assessments/adhd.html [hunk 2]
```diff
@@ -89,438 +95,495 @@ <h3 class="mm-survey-results-score"></h3>
...
+                                          <option value="age_46"> 46-55 </option>
+                                          <option value="age_56"> Above 55</option>
+                                       </select>
+                                       <br>
+                                       <br>
+                                       <label for="dropdown" id="model" style="width: 90vw;"><i
+                                             style="color: rgb(255, 0, 0); ;">*</i> Gender </label>
+                                       <select name="gender" id="gender" required>
+                                          <option value="" disabled selected>Select your option</option>
+                                          <option value="Model_S"> Male </option>
```
Comment (orchvate) [inline assessments/adhd.html:169]: Same here

File: assessments/adhd.html [hunk 3]
```diff
@@ -89,438 +95,495 @@ <h3 class="mm-survey-results-score"></h3>
...
+                                       <input id="email" type="email" name="email" placeholder="Enter here"
+                                          class="form-inputs" required> </input>
+                                       <br>
+                                       <br>
+                                       <label for="dropdown" id="model" style="width: 90vw;"><i
+                                             style="color: rgb(255, 0, 0); ;">*</i> Age </label>
+                                       <select id="dropdown"  name="age" class="required" required>
+                                          <option value="" disabled selected>Select your option</option>
+                                          <option value="age_18"> 18-25 </option>
+                                          <option value="age_26"> 26-35 </option>
```
Comment (orchvate) [inline assessments/adhd.html:158]: Values not compatible with model

File: assessments/adhd.html [hunk 4]
```diff
@@ -89,438 +95,495 @@ <h3 class="mm-survey-results-score"></h3>
...
+                                          <option value="Very Often">Very Often </option>
+                                       </select>
+                                       <br>
+                                       <br>
+                                       <label for="dropdown" id="model" style="width: 90vw;"><i
+                                             style="color: rgb(255, 0, 0); ;">*</i> 2. I have difficulty getting things
+                                          in order when I have to do a task that requires organization. </label>
+                                       <select name="q2" id="q2" class="required">
+                                          <option value="" disabled selected>Select your option</option>
+                                          <option value="Never"> Never </option>
```
Comment (orchvate) [inline assessments/adhd.html:230]: Values are not compatible with API request body

File: assessments/adhd.html [hunk 5]
```diff
@@ -89,438 +95,495 @@ <h3 class="mm-survey-results-score"></h3>
...
-                                 
-                                   </div>
-                                   
+                                    Section 2 </h3>
+                                 <h4>Part A</h4>
+                                 <h6>( Questions 1-6 )</h6>
+
+                                 <form id="adhd-survey-form-section2a">
+
+                                    <div class="input-headings">
```
Comment (orchvate) [inline assessments/adhd.html:206]: Is there individual HTML for everything? Why isnt it not rendered through a JS? Anyway too late to handle this. Needs to change later

**Comment by orchvate** [comment]
@rahullrajesh this needs some additional work. Also, I dont know why we have individual HTML for every question. Should have been one structure which repeats based on an array. Makes it easy to maintain. Too late to handle this. First please take care of the values.

Honestly I am not confident about the deployment tomorrow. I know we have to go ahead but my confidence is very low.

Cc: @ishani220396

**Comment by rahullrajesh** [comment]
okay lets see

**Pull request merged**